tasks/release_gem.rake

```diff
@@ -1,4 +1,8 @@
 Rake::Task["build"].enhance(["build:pre_check"])
+Rake::Task["build"].enhance do
+  # This syntax makes this task run after build -- see https://dev.to/molly/rake-task-enhance-method-explained-3bo0
+  Rake::Task["build:after_check"].execute
+end
 
 desc 'Checks executed before gem is built'
 task :'build:pre_check' do
```

Comment:
Note: I had on purpose also added the `world.reset` call on this one, see https://github.com/DataDog/dd-trace-rb/blob/master/tasks/release_gem.rake#L10